src/main/java/org/folio/search/service/reindex/jdbc/MergeRangeRepository.java

```diff
@@ -35,6 +36,10 @@ public abstract class MergeRangeRepository extends ReindexJdbcRepository {
 
   private static final String SELECT_MERGE_RANGES_BY_ENTITY_TYPE = "SELECT * FROM %s WHERE entity_type = ?;";
 
+  private static final String SELECT_FAILED_MERGE_RANGES = """
+    SELECT * FROM %s WHERE status = 'FAIL' and tenant_id IN (%s);
+    """;
+
```

Comment:
What is the purpose of having the tenan_id condition here? What cases are covered by it?